packages/atlas/src/components/_nft/NftWidget/NftWidget.tsx

```diff
@@ -126,14 +126,25 @@ export const NftWidget: React.FC<NftWidgetProps> = ({
         <Banner id="" dismissable={false} icon={<SvgAlertsInformative24 />} {...{ title, description }} />
       </GridItem>
     )
-    const WithdrawLastBit = ({ secondary }: { secondary?: boolean }) => (
-      <GridItem colSpan={buttonColumnSpan}>
-        <ButtonGrid data-size={size}>
-          <Button variant={secondary ? 'secondary' : undefined} fullWidth size={buttonSize}>
-            Withdraw last bid
-          </Button>
-        </ButtonGrid>
-      </GridItem>
+    const WithdrawLastBid = ({ secondary }: { secondary?: boolean }) =>
+      bidFromPreviousAuction ? (
+        <>
+          <GridItem colSpan={buttonColumnSpan}>
+            <Button variant={secondary ? 'secondary' : undefined} fullWidth size={buttonSize}>
+              Withdraw last bid
+            </Button>
+            <Text as="p" margin={{ top: 2 }} variant="t100" secondary align="center">
+              You bid {formatNumberShort(Number(bidFromPreviousAuction?.amount))} tJOY on{' '}
+              {formatDateTime(bidFromPreviousAuction.createdAt)}
```

Comment:
The app is breaking and I can't do anything when I go to the video with an open auction with my bid added. I'm getting: 
![image](https://user-images.githubusercontent.com/51168865/160823150-a9f6638b-d5d8-4acf-8d57-c32bd9c206cf.png)
I assume that it comes from here.